test/foundry/dapp/Application.t.sol

```diff
@@ -49,6 +49,7 @@ contract ApplicationTest is ERC165Test {
     IInputBox _inputBox;
     IPortal[] _portals;
     LibEmulator.State _emulator;
+    SafeERC20Transfer _safeERC20Transfer;
```

Comment:
I'd switch these two.
Note that we're first defining contracts (application, input box, portals, etc.) and then variables like emulator state, application owner, etc. So `SafeERC20Transfer` should be grouped with the contracts.